pkg/types/types.go

```diff
@@ -1047,7 +1047,7 @@ func NewParagraph(lines []interface{}, attributes interface{}) (Paragraph, error
 	if err != nil {
 		return Paragraph{}, errors.Wrapf(err, "failed to initialize a Paragraph element")
 	}
-	// log.Debugf("initializing a new paragraph with %d line(s) and %d attribute(s)", len(lines), len(attrs))
+	// log.Debugf("initializing a new Newparagraph with %d line(s) and %d attribute(s)", len(lines), len(attrs))
```

Comment:
this change could be reverted, but it's not a big deal since commented out anyways ;)

Reply:
lol.  I don't know that came in.

Reply:
I'm fixing this.